tests/integration/targets/cpanm/tasks/main.yml

```diff
@@ -0,0 +1,46 @@
+# (c) 2020, Berkhan Berkdemir
+
+# This file is part of Ansible
```

Comment:
Please use the short GPLv3+ banner instead of the long one:
```
# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
```